src/components/Box/index.tsx

```diff
@@ -1,54 +1,50 @@
-import { Theme, css } from '@emotion/react'
-import styled from '@emotion/styled'
+import { css } from '@emotion/react'
+import styled, { StyledComponent } from '@emotion/styled'
 import { x } from '@xstyled/emotion'
 import PropTypes from 'prop-types'
 import React, {
   ElementType,
-  FunctionComponent,
-  ReactNode,
+  HTMLAttributes,
+  RefAttributes,
   forwardRef,
 } from 'react'
 
-const borderedStyles = ({ theme }: { theme: Theme }) => css`
-  padding: ${theme.space['3']};
-  border-radius: ${theme.radii.default};
-  border: 1px solid ${theme.colors.gray350};
-`
-
-const StyledBox = styled(x.div, {
-  shouldForwardProp: prop => !['bordered'].includes(prop.toString()),
-})<{ bordered?: boolean }>`
-  ${({ bordered }) => (bordered ? borderedStyles : null)}
-`
-
 type BoxProps = {
   bordered?: boolean
   width?: number | string
   height?: number | string
-  children?: ReactNode
-} & XStyledProps &
-  React.HTMLAttributes<HTMLElement>
+}
 
-const forwardType = forwardRef<Element, BoxProps>(() => null)
+const StyledBox = styled(x.div, {
+  shouldForwardProp: prop => prop !== 'bordered',
+})<BoxProps>`
+  ${({ bordered, theme }) =>
+    bordered
+      ? css`
```

Comment:
Do we need to put styled and css ? This can be useful when there are some conditional styles but here, is the import worse ??? 😅